src/app/search-results/search-results.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="card my-5">
+    <div class="card-body">
+        <div class="search-form">
+                <input #searchTerm (keyup.enter)="searchByKeyword(searchTerm.value, filterBy.value)" 
```

Comment:
I had a little trouble getting this to work initially since I wasn't expecting to have to click "search" if I did not put in a search term. In other words, I expected to be able to just adjust the filter and see results, without adding a search term or clicking search. I'll try to play around with it. It works! It just may not be the best user experience.